src/plugins/share/public/url_service/redirect/components/empty_prompt.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as React from 'react';
+
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ChromeDocTitle } from '@kbn/core-chrome-browser';
+import { NotFoundPrompt } from '@kbn/shared-ux-prompt-not-found';
+
+const defaultTitle = i18n.translate('share.urlService.redirect.components.Error.title', {
+  defaultMessage: 'Unable to open URL',
+  description:
+    'Title displayed to user in redirect endpoint when redirection cannot be performed successfully.',
+});
+
+const defaultBody = i18n.translate('share.urlService.redirect.components.Error.body', {
+  defaultMessage:
+    'The stored URL for this link could not be opened. This could be due to lost Kibana Saved Object data.',
```

Comment:
I think the text could be improved. 
I like the text from the screenshot better https://github.com/elastic/kibana/pull/171679#issuecomment-1828160476


Also the URL might be broken because it just was incorrect (e.g. mistaken when manually typed or not fully copied)

Reply:
Maybe something like `Sorry, the object you're looking for can't be found at this URL. It might
have been removed or maybe it never existed.`